mne/bem.py

```diff
@@ -1090,10 +1090,17 @@ def _fit_sphere_to_headshape(info, dig_kinds, *, verbose=None):
     o_mm = origin_head * 1e3
     o_d = origin_device * 1e3
     if np.linalg.norm(origin_head[:2]) > 0.02:
-        warn(
+        msg = (
             f"(X, Y) fit ({o_mm[0]:0.1f}, {o_mm[1]:0.1f}) "
             "more than 20 mm from head frame origin"
         )
+        if dig_kinds == "auto":
+            logger.info(msg)
+            logger.info("Trying again with all digitization points.")
+            return _fit_sphere_to_headshape(
+                info, dig_kinds=("extra", "eeg", "hpi", "cardinal"), verbose=verbose
+            )
+        warn(msg)
```

Comment:
Technically this is a backward incompatible change, so needs to be considered a bugfix with an entry in the `bug` section. For that, we'd want to be fairly certain it's better in 95%+ of cases. For example, anyone using the default `origin="auto"` in `maxwell_filter` who formerly got a warning will now have their `origin` change in 1.11 (assuming they have extra dig points). I am not sure we want to do this... but maybe it's okay?

@wmvanvliet WDYT?